pylint/checkers/imports.py

```diff
@@ -264,48 +265,48 @@ class ImportsChecker(BaseChecker):
     else:
         deprecated_modules = ('optparse', 'tkinter.tix')
     options = (('deprecated-modules',
-                {'default' : deprecated_modules,
-                 'type' : 'csv',
-                 'metavar' : '<modules>',
-                 'help' : 'Deprecated modules which should not be used,'
-                          ' separated by a comma'}
-               ),
+                {'default': deprecated_modules,
+                 'type': 'csv',
+                 'metavar': '<modules>',
+                 'help': 'Deprecated modules which should not be used,'
+                         'separated by a comma'}
+                ),
                ('import-graph',
-                {'default' : '',
-                 'type' : 'string',
-                 'metavar' : '<file.dot>',
-                 'help' : 'Create a graph of every (i.e. internal and'
-                          ' external) dependencies in the given file'
-                          ' (report RP0402 must not be disabled)'}
-               ),
+                {'default': '',
+                 'type': 'string',
+                 'metavar': '<file.dot>',
+                 'help': 'Create a graph of every (i.e. internal and'
+                         ' external) dependencies in the given file'
+                         '(report RP0402 must not be disabled)'}
+                ),
                ('ext-import-graph',
-                {'default' : '',
-                 'type' : 'string',
-                 'metavar' : '<file.dot>',
-                 'help' : 'Create a graph of external dependencies in the'
-                          ' given file (report RP0402 must not be disabled)'}
-               ),
+                {'default': '',
+                 'type': 'string',
+                 'metavar': '<file.dot>',
+                 'help': 'Create a graph of external dependencies in the'
+                         ' given file (report RP0402 must not be disabled)'}
+                ),
                ('int-import-graph',
-                {'default' : '',
-                 'type' : 'string',
-                 'metavar' : '<file.dot>',
-                 'help' : 'Create a graph of internal dependencies in the'
-                          ' given file (report RP0402 must not be disabled)'}
-               ),
+                {'default': '',
+                 'type': 'string',
+                 'metavar': '<file.dot>',
+                 'help': 'Create a graph of internal dependencies in the'
+                         'given file (report RP0402 must not be disabled)'}
```

Comment:
The string is in continuation, so earlier line "... in the given file..." changes to "in thegiven file..." essentially removing the space between two words.

Reply:
fixed